data/seeds/002_children.js

```diff
@@ -0,0 +1,28 @@
+/*
+  ** CREATE 3 CHILDREN **
+  id       = 'not sure, need to ask what changed here'
+  parentId = int
+  name     = string
+  pin      = string
```

Comment:
Do we need all of this commented out code?

Reply:
No, it is now gone.